arm-web/2016-03-01/ResourceProvider.json

```diff
@@ -207,76 +271,6 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/publishingCredentials": {
```

Comment:
Ah I see. I remember seeing an issue posted by a node sdk customer. https://github.com/Azure/azure-sdk-for-node/issues/1909#issuecomment-254940255 . Ruslan recommended using this `GET /providers/Microsoft.Web/publishingUsers/web?api-version=2015-08-01` instead.
Btw, is the above api present for the new api-version (Just confirming)?